http_server/fns/data_fns.php

```diff
@@ -249,7 +249,7 @@ function talk_to_server($address, $port, $key, $server_function, $receive = fals
 
 
 
-//--- tests to see if a string contains obsene words ---------------------------------------
+// tests to see if a string contains obscene words
 function is_obsene($str)
```

Comment:
Fix the spelling of the function too!

Reply:
I just got in bed, so I think I'll wait until tomorrow to figure out everywhere that function is used 😂